rsconnect/bundle.py

```diff
@@ -1266,6 +1383,52 @@ def write_notebook_manifest_json(
     return exists(join(directory, environment.filename))
 
 
+def write_voila_manifest_json(
```

Comment:
This function currently supports the file-based workflow, like `write-manifest notebook`. Can it be extended to also support the directory-based workflow we have for applications, if the target provided is a directory? Probably better in a separate PR.